config/scheduler/settings.ini

```diff
@@ -93,6 +93,12 @@ pa.scheduler.core.listener.threadnumber=5
 # List of the scripts paths to execute at scheduler start. Paths are separated by a ';'.
 pa.scheduler.startscripts.paths=tools/LoadPackages.groovy
 
+# Path to boot (early starting) microservices
+pa.boot.microservices.path=dist/boot/
+
+Path to boot (early starting) microservices
```

Comment:
This seems to be a comment but there is no `#` at the beginning of the line.

Reply:
Fixed, thanks